.travis.yml

```diff
@@ -4,7 +4,7 @@ android:
   components:
     - platform-tools
     - tools
-    - build-tools-28.0.3
+    - build-tools-29.0.2
```

Comment:
@vj-codes @isabelcosta should we remove travis CI completely can we just update the version? I remember isabel saying that we have reached the limit so the tests have stopped running. I think we should remove Travis CI since we are using github actions anyways.

Reply:
@epicadk I agree with removing it completely, having both gh actions and travis doing the same work doesn't make it any advantageous either. 
@kartikeysaran please update the PR accordingly.

Reply:
Okay